spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_3_1_spec.rb

```diff
@@ -14,7 +15,7 @@
   end
 
   it "will perform a full refresh on v3.1" do
-    VCR.use_cassette("#{described_class.name.underscore}_3_1") do
+    VCR.use_cassette("#{described_class.name.underscore}_3_1", :allow_unused_http_interactions => true) do
```

Comment:
This means that "there are unused HTTP interactions remaining". This means that the tests are testing something different than the code we have. In my opinion we should not do it.

Reply:
removed.